clojure/yada/src/server.clj

```diff
@@ -0,0 +1,39 @@
+(ns server
+  (:require
+   [clojure.tools.logging :refer :all]
+   [yada.yada :as yada]))
+
+(def index
+  (yada/resource
+    {:methods
+     {:get
+      {:produces "text/plain"
+       :response ""}}}))
+
+(def postuser
+  (yada/resource
+    {:methods
+     {:post
+      {:produces "text/plain"
+       :response ""}}}))
+
+(def getuser
+  (yada/resource
+    {:parameters {:path {:name String}}
+     :methods
+     {:get
+      {:produces "text/plain"
+       :response
+       (fn [ctx]
+         (let [name (get-in ctx [:parameters :path :name])]
+           name))}}}))
```

Comment:
This resource is causing me some problems, apparently calling `nth` somehow that I don't understand.